src/main/java/edu/saigon/excercise/BT11.java

```diff
@@ -0,0 +1,26 @@
+package edu.saigon.excercise;
+
+import java.util.Scanner;
+
+public class BT11 {
+    public static void cacSoNguyenToThuocDayFibonacci(int n) {
```

Comment:
the method name should be start with verb as 'hienThiCacSoNguyenToThuocDayFibonacci'